storybook/src/docs/bestPractices/GridAndFormLayouts.stories.tsx

```diff
@@ -196,20 +196,19 @@ export const LargeFormOnAPage = {
 
 export const SimpleFormInADialog = {
     render: () => {
-        const Form = ({ id }: { id?: string }) => {
+        const Form = ({ id, mode }: { id?: string; mode?: "edit" | "add" }) => {
             const { rows, loading } = useData();
-            const editingExistingItem = Boolean(id);
 
-            if (editingExistingItem && loading) {
+            if (loading) {
```

Comment:
Was this removed on purpose? 
The point of this as to not wait for data to be loaded, when no data is being loaded...


```suggestion
            if (mode === "edit" && loading) {
```

Reply:
No, thanks for catching that!